File: LeetCode_Record/2351.第一次出现两次的字母.java
class Solution {
    public char repeatedCharacter(String s) {
        boolean letters[] = new boolean[26];
        for(int i = 0; i < 26; i++){
            int c = s.charAt(i) - 'a';
            if(letters[c])
                return s.charAt(i);
            letters[c] = true;
        }
        return 'a';
    }
}

File: LeetCode_Record/2351.第一次出现两次的字母.cpp
#include <string>
using namespace std;

class Solution {
public:
    char repeatedCharacter(string s) {
        bool letter[26] = {false};
        for(auto& i : s){
            int c = i - 'a';
            if(letter[c])
                return i;
            letter[c] = true;
        }
        return 'a';
    }
};